thoth/user_api/api_v1.py

```diff
@@ -731,6 +732,11 @@ def list_python_package_versions(
     page_count = ceil(entries_count / per_page)
     start_offset = _compute_offset(page=page, per_page=per_page, page_count=page_count)
 
+    if order_by and order_by in ["ASC", "DESC"]:
+        page_count = 1
+        start_offset = 0
+        per_page = entries_count
```

Comment:
You can pass directly `count=None` which will retrieve all the results in the database.